nucliadb/src/nucliadb/search/search/query_parser/models.py

```diff
@@ -172,7 +172,7 @@ class Date:
 
 class CatalogQuery(BaseModel):
     kbid: str
-    query: str
+    query: Optional[search_models.CatalogQuery]
```

Comment:
Query can be None when you simply want to paginate all, right? Just like before it was query = ""?

Reply:
Yes, the query planner converts this. I just feel None is clearer than an empty string.